code/controllers/subsystem/job.dm

```diff
@@ -86,6 +86,14 @@ SUBSYSTEM_DEF(job)
 	Debug("AR has failed, Player: [player], Rank: [rank]")
 	return 0
 
+/datum/controller/subsystem/job/proc/FreeRole(rank)
+	if(!rank)
+		return
+	Debug("Freeing role: [rank]")
```

Comment:
Will need the debug code gone: yaya, nono?

Reply:
Not that sort of a debug. This is just for logging. It stays. >:C